src/timeseriesflattenerv2/_process_spec.py

```diff
@@ -61,21 +49,22 @@ def _null_values_outside_lookwindow(
 
 
 def _slice_frame(
-    timedelta_frame: TimedeltaFrame,
-    lookdistance: LookDistance,
-    column_prefix: str,
-    value_col_name: str,
+    timedelta_frame: TimedeltaFrame, lookperiod: LookPeriod, column_prefix: str, value_col_name: str
 ) -> TimeMaskedFrame:
-    new_colname = f"{column_prefix}_{value_col_name}_within_{abs(lookdistance.days)}_days"
+    # TODO: #436 base suffix on the type of timedelta (days, hours, minutes)
 
     timedelta_col = pl.col(timedelta_frame.timedelta_col_name)
 
-    is_lookbehind = lookdistance < dt.timedelta(0)
+    is_lookbehind = lookperiod.first < dt.timedelta(0)
+    new_colname_prefix = f"{column_prefix}_{value_col_name}_within_"
 
     # The predictor case
     if is_lookbehind:
-        after_lookbehind_start = lookdistance <= timedelta_col
-        before_prediction_time = timedelta_col <= dt.timedelta(0)
+        new_colname = (
+            new_colname_prefix + f"{abs(lookperiod.last.days)}_to_{abs(lookperiod.first.days)}_days"
+        )
+        after_lookbehind_start = timedelta_col >= lookperiod.first
```

Comment:
```suggestion
        after_lookbehind_start = lookperiod.first <= timedelta_col
```

Just a small readability preference ❤️